### ISSUE TITLE
ESN Inbox expand/collapse button is not keyboard accessible

### ISSUE THREAD

**Issue by pinterface** [dreamwidth/dw-free#1808]
A user complained about this in http://www.dreamwidth.org/support/see_request?id=29428

I don't have JAWS, but I can confirm that the button works via mouse, but does not work via keyboard.  I suspect they're running into that, and not a JAWS-specific issue.

Technically, it's also a link, but the link doesn't do anything useful (it reloads the page, but doesn't change anything).  Only users running without JS (hi!) are likely to notice that aspect, though.

**Comment by rahaeli** [comment]
Oh wow, embarrassed that I missed the user had replied back to that one so long ago. Eep!

If you want to run it through actual JAWS testing, @deborahgu has it installed and has gotten pretty good at testing with it. But I agree that this is likely to be the culprit.

**Comment by kareila** [comment]
Hey @pinterface you can decide if you want to leave this open for the non-JS case and if so, if you want to keep it claimed or leave it for someone else.

**Comment by pinterface** [comment]
I think it makes sense to close this.  Yes, having the Inbox work without JS would be nice, but there are already two open bugs about redoing it entirely (#1094 and #1224), so it seems silly to expend very much effort on the current design.

**Comment by kareila** [comment]
Fair enough :)
